src/pages/en/install/auto.md

```diff
@@ -36,62 +34,23 @@ yarn create astro
-:::
-
-### Install Dependencies (optional)
-The wizard will offer to run an `install` command for you at this time, which is optional.
+1. **Setting a new project directory.**
+1. **Selecting a starter project template.**
+1. **Optionally installing your new npm dependencies.**
+2. **Optionally initializing a new git repo for your project.**
 
-:::caution
-If you do not do so at this time, you will need to [install dependencies](/en/install/auto/#2-install-dependencies) after the wizard has finished, before starting your project.
-:::
+You do not need to create a new directory for your project before running the wizard. As a part of setup, the `create-astro` wizard will ask you where your new project should live and will create a new folder for it if needed.
 
-### Install any Official Astro Integrations (optional)
-You will be given the option at this time to add any [additional UI frameworks](/en/core-concepts/framework-components/) (React, Svelte, Vue, Solid, Preact, Lit) and then other Astro official integrations (Tailwind, Partytown, Sitemap) by running `astro add --yes`
+Before running, some package managers may prompt you to confirm that you want to install the latest version of `create-astro` (`create-astro@latest`). This is expected, and it is okay to accept this prompt so that the `create-astro` wizard can run on your machine.
 
+## 2. Install NPM Dependencies
 
-To select which Astro integrations, if any, you would like to include in your project, use the arrow keys (up and down) to navigate and the space bar to toggle between selected states. You can select multiple items at once, or you can continue without selecting any integrations.
+If you already installed your dependencies during `create-astro` setup, then you can skip this and move on to the next step.
 
+Astro is distributed via the [npm ecosystem](https://www.npmjs.com/package/astro), so you will need to install it using `npm` or another npm package manager of your choice. 
 
-When you are satisfied with your selection, press return (enter) to submit.
-
-:::note
-These integrations, and any [Astro community integrations](https://astro.build/integrations/), can also be added later by following the instructions in our [integrations guide](/en/guides/integrations-guide/).
-:::
-
-
-After selecting your integrations to add, you should see the following terminal message notifying you of the changes that `create-astro` will apply to your project's `astro.config.mjs`:
-
-```bash
-Astro will make the following changes to your config file:
-```
-
-This message should show that your chosen integrations have been successfully added to your project configuration. (If not, you can always add them manually later.)
-
-
-### Initialize a `.git` Repository (optional)
-
-At this final step, you can choose to initialize a git repository in your new directory. This is optional, but useful if you plan to use the tool [Git](https://git-scm.com/) for your project.
```

Comment:
I'm really glad you brought this up since this is the main thing that I'm excited about with this PR! I agree on the primary goal of making sure that we are catering to beginners, including someone who doesn't know what Git is. 

But, I'm introducing a second goal here in this PR: The thing that we're documenting (in this case, `create-astro`) should also be designed with this same user in mind. If that user doesn't know what Git is, then the prompt will confuse them, and they'd have to know to stop what they're doing, know to leave the terminal and switch to their web browser, know to read this section, and then they'd feel better. That's better than nothing but also not ideal.

Instead, I spent the time in https://github.com/withastro/astro/pull/3715 to make sure the tool itself was super beginner friendly. You can watch the video to see it in action, but now if you say no to the prompt it says _"Sounds good! You can come back and run **git init** later."_ I also just updated the question to include the useful bit you added here, about why someone may want to say "yes" to the prompt.

Does that make sense? Docs shouldn't be used to paper over confusing bits of the thing that its documenting, when we can focus on improving the thing to be less confusing. Not only is it a better flow for the user, but it also covers the user who gives up instead of flipping back to the docs.

Watching you all have to document how the multiselect worked was another really good example of this!

Reply:
Perfect! Yes this makes sense. Really happy to see this feed back into the CLI design. Thank you 💜